infrastructure/storage/tests/lmdb.rs

```diff
@@ -0,0 +1,247 @@
+// Copyright 2019. The Tari Project
```

Comment:
All the tests can be improved by trying to read the values again. The tests currently assume that the lmdb will fail in such a way that an error is thrown if it fails.

Reply:
I'm not clear on what you're saying here. Could you elaborate?

Reply:
The following tests only write data to the db:
-  fn multi_thread() 
- fn multi_thread_writes()

After the data was written there is no check to ensure that the data is actually in the db. All that is actually tested is that write does not panic, or return an error.

Reply:
It is checked in multi_thread, actually, but good catch on mtw -- added